lib/rubocop/cop/naming/rescued_exceptions_variable_name.rb

```diff
@@ -150,6 +153,13 @@ def message(node)
           preferred_name = preferred_name(offending_name)
           format(MSG, preferred: preferred_name, bad: offending_name)
         end
+
+        def check_shadowed_variable_name(node)
```

Comment:
Can you rename to predicate method?

```suggestion
        def shadowed_variable_name?(node)
```